go-controller/pkg/libovsdb/ops/model.go

```diff
@@ -489,6 +489,39 @@ func buildFailOnDuplicateOps(c client.Client, m model.Model) ([]ovsdb.Operation,
 	return c.WhereAny(m, cond).Wait(ovsdb.WaitConditionNotEqual, &timeout, m, field)
 }
 
+func getNonZeroFields(model model.Model) []interface{} {
```

Comment:
This will not work, if user want to reset a field it cannot since it will never be updated.

Reply:
you are totally right.

Reply:
Well, it would work, but we would have to be explicit and set it to true or false always. Choose your poison.

Reply:
Meaning, is not as implicit as having it always in `getAllMutableFileds`